src/shims/ApiCompatBaseline.uap.netstandard20.txt

```diff
@@ -1 +1,8 @@
-Total Issues: 0
+Compat issues with assembly mscorlib:
```

Comment:
It is because of https://github.com/dotnet/standard/pull/332 where we preemptively added some type-forwards into the mscorlib shim that ships with NS2.0. It isn't really an issue and will naturally fallout once we decide to make .NET Core compatible with .NET 4.7 .